qvisor/src/arch/x86_64/mod.rs

```diff
@@ -12,4 +12,166 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod context;
\ No newline at end of file
+use kvm_bindings::{kvm_regs, CpuId};
+use kvm_ioctls::Kvm;
+use std::sync::mpsc::Sender;
+use std::os::unix::io::{AsRawFd, FromRawFd};
+use std::convert::TryInto;
+use libc::ioctl;
+use spin::{Mutex, MutexGuard};
+
+use crate::QUARK_CONFIG;
+use crate::arch::vCPU;
+use crate::runc::runtime::vm::{Dump, ClearDump};
+use crate::qlib::common::{KERNEL_FLAGS_SET, Error};
+use crate::qlib::linux_def::MemoryDef;
+
+pub mod context;
+pub mod vm;
+
+#[derive(Debug)]
+pub struct x86_64vCPU {
```

Comment:
The naming of x86_64vCPU struct looks kinda uncanny to me .. is the cargo linter ok with this?

Reply:
> is the cargo linter ok with this?
Specific on this, yes, but in general, no ;)
>naming looks uncanny to me
Happy to change it with any suggestion.